### ISSUE TITLE
Various issues, mistakes and/or bugs *SPOILERS*

### ISSUE THREAD

**Issue by MinmusxMinmus** [07th-mod/tsumihoroboshi#14]
This post was created to sum up all the small issues I have found in the patched game.

1 - Visible code

[07/18/18]
![image](https://user-images.githubusercontent.com/41398315/42910047-5f13a708-8ae6-11e8-9e47-aeb9698134b0.png)
(In the save file text preview)

2 - Long dialog that doesn't fit the updated UI

[07/18/18]
![image](https://user-images.githubusercontent.com/41398315/42910099-86cf169c-8ae6-11e8-84be-8d54453ba37f.png)

[07/18/18]
![image](https://user-images.githubusercontent.com/41398315/42910109-8c69d6b4-8ae6-11e8-8046-ff896894f786.png)

[07/18/18]
![image](https://user-images.githubusercontent.com/41398315/42912370-55457e5a-8aef-11e8-90bb-ac8b9f621f5b.png)

[07/19/18]
![image](https://user-images.githubusercontent.com/41398315/42968409-6e9f3a9e-8ba3-11e8-98c8-1b96df06db98.png)
(In addition to this, a small missing space is noticeable at "with theculture-bound")

[07/20/18]
![image](https://user-images.githubusercontent.com/41398315/43004447-19cd888a-8c30-11e8-9183-ebda988e068e.png)

3 - Wrong character

[07/18/18]
![image](https://user-images.githubusercontent.com/41398315/42910137-a3f692a4-8ae6-11e8-8b91-c5b5bcdd634a.png)
(P.S: The orange underline signifies Rena dialog, while the grey underline signifies Rena's dad dialog)

[07/20/18]
![image](https://user-images.githubusercontent.com/41398315/42999023-8e99c700-8c1c-11e8-8247-75cf9e185e02.png)
(P.S: According to the voices, the orange underlines signify Rena's dialog, and the grey underlines are Ooishi's dialog)

4 - Small grammar/spelling mistakes

[07/18/18]
![image](https://user-images.githubusercontent.com/41398315/42910189-d62547b6-8ae6-11e8-9be5-783d745795aa.png)
(A "have" is missing in the red rectangle)

[07/18/18]
![image](https://user-images.githubusercontent.com/41398315/42912081-dca6ff38-8aed-11e8-860b-a13996935998.png)
(On Ooishi's last dialog, a space should be added between the dot and "Please")

[07/19/18]
![image](https://user-images.githubusercontent.com/41398315/42967036-495c58d8-8b9f-11e8-9bc9-929a4e1635ba.png)
On Mion's dialog, a spelling mistake in "Minster"

[07/19/18]
![image](https://user-images.githubusercontent.com/41398315/42967941-23eec628-8ba2-11e8-8318-b8193810c559.png)
On the last thought box, an "I" is missing in "If stayed here"

[07/19/18]
![image](https://user-images.githubusercontent.com/41398315/42974152-57e4f5c4-8bb6-11e8-993f-5dbb2209220d.png)
On Mion's dialog, a "to" is missing in "What made you want tell us?"

-MxM

**Comment by Chibiki** [comment]
Also Rina has no lip sync; at least where I’m reading, which is the Angel Mort eating competition very early into the story

**Comment by DoctorDiablo** [comment]
Hijacking this thread to report this bit of weirdness:
![image](https://user-images.githubusercontent.com/20390474/42913326-69395526-8ac2-11e8-8e18-3f21b6086fc1.png)

**Comment by enumag** [comment]
Don't hijack an existing thread guys pls. It makes it difficult to track what is fixed and what isn't.

**Comment by enumag** [comment]
@MinmusxMinmus All of your issues has been fixed, thanks for reporting.

@Chibiki Needs more information, like specific lines that didn't have lipsync. Create a separate issue for such problems.

@DoctorDiablo This seems to be a rare bug of the engine when it doesn't clear a layer before rendering a new sprite. I have seen it before for Satoko but it never happened again when I tried to reproduce it. I checked the sprites around this line and they are correct.

Closing this issue. Open a new one if you see any more problems.
